src/pages/guest/page/GuestMyClass/GuestMyClass.tsx

```diff
@@ -13,9 +15,16 @@ import {
 import { useState } from 'react';
 import { GUEST_MY_CLASS_DATA } from 'src/constants/mocks/guestMyClassCardData';
 import { GuestMyClassEmptyView, MoimCard } from '@pages/guest/components';
+import { useFetchGuestParticipate } from '@apis/domains/moim/useFetchGuestParticipate';
+import { useAtom } from 'jotai';
+import { userAtom } from '@stores';
 
 const GuestMyClass = () => {
   const [activeTab, setActiveTab] = useState<'신청한' | '참가한'>('신청한');
+  const [user] = useAtom(userAtom);
+  const { data } = useFetchGuestParticipate(user.guestId); //참가한 조회
+
+  console.log('gusetMyClassData', data);
```

Comment:
```suggestion
  const { data: gusetMyClassData } = useFetchGuestParticipate(user.guestId); //참가한 조회
  
  console.log('guestMyClassData': guestMyClassData)
```

이렇게 쓰면 data 접근할 때 gusetMyClassData 이름으로 쓸 수 있대요!!
여기는 내용이 많지 않아서 data로 해도 너무 잘 보이는데!! 혹시 나중에 길어지거나 복잡해지면 data 대신에 이름 지정해주는 것도 너무 좋을 것 같아용!!

아까 배움 , , , , 😋

Reply:
잊어먹고 있던 부분이었는데 리마인드 감사합니다 ㅎㅎ!
여기 브랜치 말고 merge 후에 실제 로직 구현할 때에 바로 적용할게요 :)